gradle.properties

```diff
@@ -27,8 +27,8 @@ developmentEnvironmentUserName = Developer
 # version in @Mod([...], version = VERSION, [...])
 # Leave these properties empty to skip individual token replacements
 replaceGradleTokenInFile = Reference.java
-gradleTokenModId =
-gradleTokenModName =
+gradleTokenModId = GRADLETOKEN_MODID
+gradleTokenModName = GRADLETOKEN_MODNAME
```

Comment:
Please don't. While this *probably* won't change anything, there is no reason to do this either and each time you *do* apply these tokens, you have a small chance of it breaking